tests/create.test.js

```diff
@@ -0,0 +1,34 @@
+import test from './async-test';
+import { resolve } from 'path';
+import { create } from './lib/cli';
+import lsr from './lib/lsr';
+import { setup, clean } from './lib/output';
+
+const listTemplate = async dir => await lsr(resolve(__dirname, '../examples', dir), ['.gitkeep', 'package.json']);
+const listOutput = async dir => await lsr(dir, ['.gitkeep', 'package.json']);
+
+test('preact create - before', async () => {
+	await setup();
```

Comment:
I separated them because it the 'main' test fails then they will be called for sure.

It doesn't matter for `create` or `build` that much (they could be handled inside tests and in `test.onFinish` hook), but stuff with chrome get's trickier as we must ensure service worker is unregistered after tests because on non-CI environment service workers could provide false positives (I've tried using profile flag to ensure clean state to no avail).

I thought that consistency between all tests would be easier to maintain & extend.